docs/home-manager.adoc

```diff
@@ -42,3 +42,36 @@ Then we should be able to use the given module. E.g.
   };
 }
 ----
+
+=== Custom vim plugins
+
+It's possible to add custom vim plugins by using the startPlugins and lua DAG settings. First we install the plugin by adding it to startPlugins, I'll be using the nvim-surround plugin for example. This process should be very similar for nixos installations.
```

Comment:
Remember that home-manager is a cross-distro program, so the process is similar not only for a nixos installation, but quite literally any other installation.

"This example uses nvim-surround, but the process will be similar for other plugins as well".

Reply:
Ah okay, sorry. I copied it from your message 'do mention the process will be extremely similar for nixos installations' xD.